apps/services/license-api/src/app/modules/license/clients/firearmLicense/firearmLicenseMapper.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  FirearmProperty,
+  FirearmPropertyList,
+  LicenseInfo,
+} from '@island.is/clients/firearm-license'
+import format from 'date-fns/format'
+import { format as formatSsn } from 'kennitala'
+
+export const formatNationalId = (nationalId: string) => formatSsn(nationalId)
```

Comment:
This looks like unnecessary reexport of kennitala's format method, which should just be used instead of this.